configure.ac

```diff
@@ -147,6 +147,7 @@ AS_IF([test "x$enable_directwrite" != xno], [
     AC_COMPILE_IFELSE([
         AC_LANG_PROGRAM([[#include <windows.h>]], [[;]])
     ], [
+        LIBS="$LIBS -lgdi32"
```

Comment:
For some reason I had the impression that VLC didn’t use our `configure` (and hence I wasn’t sure whether it was possible at all to use it for WinRT builds), but looking at their code again, it seems it totally does.

In that case, it would seem ideal if `configure.ac` could automatically add either `-lgdi32` or `-ldwrite` depending on whether the target it Win32 or WinRT. But I’ll need to do more investigation to figure out how.

Reply:
Fixed: `configure` now detects whether we’re targeting Win32/desktop or WinRT/app and links to either `gdi32` or `dwrite` accordingly.